packages/dapp-svelte-wallet/react-ui/src/components/CardItem.jsx

```diff
@@ -1,9 +1,9 @@
 import './CardItem.scss';
 
-const CardItem = ({ children }) => {
+const CardItem = ({ children, hideDivider }) => {
   return (
     <div className="CardItem">
-      <div className="Divider" />
+      {!hideDivider && <div className="Divider" />}
```

Comment:
I have a mild preference for:

```suggestion
      {hideDivider || <div className="Divider" />}
```

Reply:
SGTM, done